docs/get-started.rst

```diff
@@ -182,6 +182,15 @@ Make sure you have the system requirements listed in the
     cd kinto/
     make serve
 
+During the installation, you will be asked which backend you would like to use:
+
+::
+
+    $ Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory)
+
+If you don't know, just push "enter" to choose the default Memory backend.
+You can always change your backend selection 
+`later on <https://kinto.readthedocs.org/en/latest/configuration/settings.html#backends>``.
```

Comment:
I believe this link is incorrect. rest links needs to end with an underscore. Something like:

```rst
`later on <https://kinto.readthedocs.org/en/latest/configuration/settings.html#backends>`_
```

You also have an extra backtick there (at the end of the line).